types/network/network.go

```diff
@@ -31,3 +40,26 @@ type EndpointSettings struct {
 	GlobalIPv6PrefixLen int
 	MacAddress          string
 }
+
+// CleanOperationalData resets the operational data for this endpoint
+func (es *EndpointSettings) CleanOperationalData() {
```

Comment:
do we need this function here? I'd rather not add methods that are not part of the api. This can easily be implemented in docker.